src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/create_window_request.cc

```diff
@@ -166,6 +166,26 @@ void CreateWindowRequest::Run() {
         (*message_)[strings::msg_params]
                    [strings::duplicate_updates_from_window_id]
                        .asInt();
+
+    const auto default_window_id =
+        mobile_apis::PredefinedWindows::DEFAULT_WINDOW;
+
+    if (default_window_id == duplicate_updates_from_window_id) {
+      const auto current_stringified_layout =
+          application->window_layout(default_window_id);
+      const auto web_view_layout = mobile_apis::PredefinedLayout::WEB_VIEW;
```

Comment:
@AByzhynar we really need to create `web_view_layout` variable?
` if (mobile_apis::PredefinedLayout::WEB_VIEW == current_app_layout)` looks readable too

Reply:
@ydementieiev I don't mind.

Reply:
@ydementieiev Done in 04f92c26f1870a77e6cd6273b7e060add612d6b2